app/views/payments/add.html.erb

```diff
@@ -0,0 +1,39 @@
+<% content_for :title, "Saldo opwaarderen voor #{@user.name} - SOFIA" %>
+
+<%= content_tag :div, id: 'payment-add', class: 'container footer-padding', data: {user_credit: @user.credit, payment_amount: @payment.amount} do %>
+  <div class="container footer-padding">
+    <div class="row py-4">
+      <div class="col-12">
+        <h1>
+          Saldo opwaarderen voor <%= @user.name %>
+        </h1>
+
+        <%= simple_form_for(@payment, wrapper: :horizontal_form) do |f| %>
+          <div class="col-12">
+            <p>
+              Voer hier het bedrag in dat je wilt inleggen. Hierna zal je worden doorgestuurd naar de betaalomgeving.
+              Let op: Het bedrag moet gescheiden worden door een punt. Gebruik geen komma!
+            </p>
+            <%= f.input :amount, as: :float, value: 10.00, input_html: {"v-model": "paymentAmount"}, required: true, label: 'Bedrag'  %>
```

Comment:
Why does it have a default value (of 10) if it also has a v-model that changes it immediately after Vue initialization?